requirements.txt

```diff
@@ -1,3 +1,4 @@
+build
```

Comment:
The way we've done this in bio2zarr is to add a tests optional dependencies. So, no more requirements.txt anywhere

Reply:
Yeah, I'm in two minds about that—mainly because I forget the odd `'[.dev]` syntax for pip every time I use it.